kurtosis-devnet/justfile

```diff
@@ -95,6 +99,25 @@ pectra-devnet: (devnet "pectra.yaml")
 # Isthmus devnet
 isthmus-devnet: (devnet "isthmus.yaml")
 
+# Celo Isthmus devnet
+celo-isthmus-devnet:
+    ./setup-local-repos.sh Kourin1996/celo-rebase-13-contracts-devnet celo-rebase-13-no-HF-overrides
```

Comment:
What cnahges do those branches include? Should we merge those first?

Reply:
`Kourin1996/celo-rebase-13-contracts-devnet` is a branch for op-deployer and foundry scripts. This branch is forked from a different branch [celo-contracts/v3.0.0--1](https://github.com/celo-org/optimism/tree/celo-contracts/v3.0.0--1).

`celo-rebase-13-no-HF-overrides` is a branch of op-geth forked from `celo-rebase-13. It doesn't include overriding holocene and isthmus activation times.